.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+    "search.exclude": {
+        "**/vendor": true,
```

Comment:
Then you should add it to your user `search.exclude`. This doesn't change how that works, right?